.agent/memory.md

```
@@ -291,4 +291,9 @@ COMPOSE_PROJECT_NAME=coneshare docker-compose exec frontend npm test -- --run sr
 - **Context/Implication:** The documents list reset to default sorting upon page reloads. In contrast, Datarooms rely on manual index/position order for due diligence workflows.
 - **Resolution/Action:** Added optional `storageKey` persistence to `useSortedList` (`frontend/src/hooks/useSortedList.js`) with validation and fallback to defaults. Enabled `coneshare_documents_sort` on `DocumentsPage` (`frontend/src/pages/DocumentsPage.jsx`) while intentionally keeping `DataroomPage` (`frontend/src/pages/DataroomPage.jsx`) unpersisted to preserve canonical position ordering on reloads.
 
+### 2026-08-16 Session Entry
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)